app/services/move_project_to_channel.rb

```diff
@@ -1,13 +1,21 @@
 class MoveProjectToChannel
-  def initialize(project_params,channel_params)
-    @project = Project.find(project_params)
-    @channel = Channel.find(channel_params)
+  def initialize(project_id, channel_id)
+    @project = Project.find_by(id: project_id)
+    @channel = Channel.find_by(id: channel_id)
+  end
+
+  def self.call(project_id, channel_id)
+    new(project_id, channel_id).call
   end
 
   def call
```

Comment:
What do you think to rename this method to `perform` and return `true` if the project had its channel updated and false if not? I think this way the service shares some responsibilities with the controller, you should let the controller handle the message which will be returned for the user. Ah, and remember to create the translate keys for this operation.